scripts/run_mypy_checks.py

```diff
@@ -769,37 +774,65 @@ def install_third_party_libraries(skip_install):
         install_third_party_libs.main()
 
 
-def get_mypy_cmd(files):
+def get_mypy_cmd(files, ci_check):
     """Return the appropriate command to be run.
 
     Args:
         files: list(list(str)). List having first element as list of string.
+        ci_check: bool. Whether the test is for CI.
 
     Returns:
         list(str). List of command line arguments.
     """
+    # TODO(#13113): Change mypy command to mypy path after Python3 migration.
+    if ci_check:
+        mypy_cmd = 'mypy'
+    else:
+        mypy_cmd = os.path.join(
+            os.getcwd(), 'third_party', 'python3_libs', 'bin', 'mypy')
     if files:
-        cmd = [MYPY_CMD, '--config-file', CONFIG_FILE_PATH] + files
+        cmd = [mypy_cmd, '--config-file', CONFIG_FILE_PATH] + files
     else:
         excluded_files_regex = (
             '|'.join(NOT_FULLY_COVERED_FILES + EXCLUDED_DIRECTORIES))
         cmd = [
-            MYPY_CMD, '--exclude', excluded_files_regex,
+            mypy_cmd, '--exclude', excluded_files_regex,
             '--config-file', CONFIG_FILE_PATH, '.'
         ]
     return cmd
 
 
-def install_mypy_prerequisites():
+def install_mypy_prerequisites(ci_check):
     """Install mypy and type stubs from mypy_requirements.txt.
 
+    Args:
+        ci_check: bool. Whether the test is for CI.
+
     Returns:
         int. The return code from installing prerequisites.
     """
-    cmd = [PYTHON3_CMD, '-m', 'pip', 'install', '-r', MYPY_REQUIREMENTS_PATH]
-    process = subprocess.call(
-        cmd, stdin=subprocess.PIPE, stdout=subprocess.PIPE)
-    return process
+    # TODO(#13398): Change MyPy installation after Python3 migration.
```

Comment:
Also, wxplain what's unexpected here which need to be fixed*

Reply:
In general we expect to have similar setup/workflow on CI, this helps us ensure that the general setup proccess is working fine if CI tests are passing. So this need to be fixed asap after the migration!

Reply:
@DubeySandeep The plan is that after python3 migration, we will look how CI checks behave. I and @vojtechjelinek tried to fix this. But the installation in CI is not creating the `bin` folder as it creates locally. 
That's why we install it globally for CI.

Reply:
I agree with Sandeep's top comment that the TODO message is unclear. What needs about the installation needs to "change"?

Reply:
Done

Reply:
Pushed it just now. What I do is that I make changes locally and reply to the comments. Then push. Then Ask for review.